streaming/text/convert/README.md

```diff
@@ -2,16 +2,71 @@
 
 To use Streaming Dataset, We first convert the dataset from its native format to MosaicML's streaming dataset format (a collection of binary `.mds` files). Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, etc.) and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all just works.
 
-## Converting C4 to streaming dataset .mds format
+Follow the below steps to convert the Natural Language Processing dataset into a streaming MDS format. Also checkout the supported [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for advanced usage.
 
-To make yourself a copy of C4, use `c4.py` as shown below with the minimalist arguments. For advanced users, you can look at the supported arguments for `c4.py` and change according to your own needs. The `c4.py` script uses the subset `en`.
 
-```
-# It downloads the 'raw format train' and 'val' split and convert to StreamingDataset format.
-# For 'val' split, it will take anywhere from 10 sec to 1 min depending on your Internet bandwidth.
-# For 'train' split, it will take anywhere from 1-to-many hours depending on bandwidth, # CPUs, etc.
-# Once the script completes, you should see a dataset folder `./my-copy-c4/val` that is ~0.5GB and
-# `./my-copy-c4/train` this is ~800GB.
+## 1. [C4: Colossal, Cleaned, Common Crawl dataset](https://huggingface.co/datasets/c4)
 
-python c4.py --out_root ./my-copy-c4
-```
+**Instructions:**
+
+1. Run the [c4.py](c4.py) script as shown below with the minimalist arguments. The script downloads the raw format `train` and `val` split from HuggingFace hub and converts to StreamingDataset MDS format into their own split directory. For advanced users, you can look at the supported arguments for [c4.py](c4.py) and change according to your own needs.
```

Comment:
```suggestion
1. Run the [c4.py](c4.py) script as shown below with the minimalist arguments. The script downloads the raw format with `train` and `val` splits from HuggingFace hub and converts to StreamingDataset MDS format into their own split directories. For more advanced use cases, please see the supported arguments for [c4.py](c4.py) and modify as necessary.
```

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/183/commits/4d242aaf5f26f73278a555a3021da6e328d0a28b.